### PR TITLE
Fix #1087, wrap FileWriter with BufferedWriter

### LINKED ISSUE #1087: FileWriter.write() should not be intensively used in loop
In the class `server/base/src/main/java/org/apache/accumulo/server/util/Admin.java`, the FileWriter.write() method is intensively used in loops. Since this method is an IO operation, it will have bad impact to the performance. As an alternative, the decorator class BufferedWriter can be used, which can significantly reduce the number of IO operations.

### PR DESCRIPTION
**Pull request by bd2019us**
Fix #1087, add wrapper for FileWriter to improve IO operation performance.

### CODE REVIEW

**Comment by milleruntime** [comment]
Thanks @bd2019us I will take a look at this today.

**Comment by milleruntime** [comment]
I am just curious... do you have a lot of tables and try printing the config where you saw a performance hit?

File: server/base/src/main/java/org/apache/accumulo/server/util/Admin.java [hunk 1]
```diff
@@ -506,7 +507,7 @@ private void printNameSpaceConfiguration(AccumuloClient accumuloClient, String n
       File outputDirectory)
       throws IOException, AccumuloException, AccumuloSecurityException, NamespaceNotFoundException {
     File namespaceScript = new File(outputDirectory, namespace + NS_FILE_SUFFIX);
-    FileWriter nsWriter = new FileWriter(namespaceScript);
+    BufferedWriter nsWriter = new BufferedWriter(new FileWriter(namespaceScript));
```
Comment (ctubbsii) [inline server/base/src/main/java/org/apache/accumulo/server/util/Admin.java]: This should be done in a try-with-resources block.

Comment (bd2019us) [reply server/base/src/main/java/org/apache/accumulo/server/util/Admin.java]: Hi @ctubbsii  , thanks for your reply. I additionally move the BufferedWriter into the resource block of try. Please check. Thanks.

**Comment by ctubbsii** [comment]
Thanks for the pull request, @bd2019us . If you're not already added, and would like to be added to https://accumulo.apache.org/people/ , feel free to open a pull request to https://github.com/apache/accumulo-website/edit/master/pages/people.md

**Pull request merged**